src/overview/design-at-a-glance.md

```diff
@@ -1,48 +1,31 @@
 # Design at a Glance
 
-The PoW+TFL consenus protocol is logically an extension of the Zcash consensus rules to introduce *trailing finality*. This is achieved by compartmentalizing the top-level PoW+TFL protocol into two *subprotocols*, one embodying most of the current consensus logic of Zcash and the TFL. These subprotocols interace through a strictly defined message-passing system called the *Subprotocol Interface*. (Remember to refer to [Terminology](../terminology.md) to clarify terms.)
+The [PoW+TFL](../terminology.md#definition-pow-tfl) consensus protocol is logically an extension of the Zcash consensus rules to introduce [trailing finality](../terminology.md#definition-trailing-finality). This is achieved by compartmentalizing the top-level PoW+TFL protocol into two [consensus subprotocols](../terminology.md#definition-consensus-subprotocols), one embodying most of the current consensus logic of Zcash and another the TFL. These protocols interact through a [hybrid construction](../terminology.md#definition-hybrid-construction), which specifies how the protocols interact, and what changes from "off-the-shelf" behavior, if any, need to be imposed on the subprotocols. Each of these components (the two subprotocols and the hybrid construction) are somewhat modular: different subprotocols or hybrid constructions may be combined (with some modification) to product a candidate [PoW+TFL](../terminology.md#definition-pow-tfl) protocol.
 
 **TODO:** Add consensus subprotocol diagram.
 
-## Subprotocols
-
-The PoW+TFL hybrid consensus consists of two interacting subprotocols:
-
-1. *PoW Subprotocol*: this subprotocol is very similar to NU5 consensus. It is a design goal of the TFL design to minimize changes to this subprotocol. Note: the shorthand "PoW" is potentially misleading, because this subprotocol is also responsible for the bulk of all supply and transaction semantic consensus rules.
-2. *TFL Subprotocol*: this is a new subprotocol which provides trailing finality via a finalizing PoS protocol.
-
-Validators must operate both subprotocols in an integrated manner. These subprotocols follow the design layed out in [Ebb-and-Flow design](../references.md#ebb-and-flow-protocols).
-
-## Design Analysis Focus Areas
-
-Analyzing this design focuses on four areas:
+## Hybrid Construction
 
-- The design of each subprotocol independently,
-- The interaction between the two subprotocols idealized as an interface between subcomponents, and
-- The whole as an integrated system.
+The [hybrid construction](../terminology.md#definition-hybrid-construction) is a major design component of the full consensus protocol which specifies how the subprotocols integrate. So far we have considered three candidates:
 
-## Consensus
+1. The implied/loosely defined hybrid construction presented at Zcon4. **TODO: Link**
+2. The [Snap-and-Chat](../terminology.md#definition-snap-and-chat) from the Ebb-and-Flow paper. **TODO: Add References and link.**
```

Comment:
```suggestion
2. The [Snap-and-Chat](../terminology.md#definition-snap-and-chat) from the [Ebb-and-Flow paper](https://eprint.iacr.org/2020/1091).
```
I don't know whether you want a more formal bibliography, but this will do for now I think.